### ISSUE TITLE
missing args files

### ISSUE THREAD

**Issue by michelaldaher** [Tostino/jpgAgent#9]
hello, we are facing an urgent issue on production after the server is restarted the args file is missing.
we need to re-add it manually in order to fix the issue 
jpgagent version: 1.3.3

**Comment by Tostino** [comment]
Hmm, there is no code in jpgagent that does anything with that file other than read it for configuration. It couldn't have deleted it. I would look at whatever process could have access to delete that file.

**Comment by michelaldaher** [comment]
Can you advise how we can troubleshoot it ?

**Comment by Tostino** [comment]
Not really, as it doesn't seem possible to be an issue with this software, and instead an issue with your server configuration which I don't have access to.

I'd look at file system corruption, but that is a wild guess at this point. The fact is, if you are saying to start jpgagent with an args file, the file needs to exist. No idea why it would disappear on you, but you need to put it back and then you should be able to start up jpgagent again.
